AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.ts

```diff
@@ -400,6 +415,17 @@ export class BookingsListComponent implements OnInit, OnDestroy {
         );
     }
 
+    private loadUsersList(): void {
+        const self = this;
+        this.videoHearingService.getUsers().subscribe(
+            (data: JusticeUserResponse[]) => {
+                this.users = data;
```

Comment:
are we showing all of the users in this dropdown list? It should only show the users in the CSO role

Reply:
it is not in the acceptance criteria